web_infrastructure/jira.py

```diff
@@ -36,7 +36,7 @@
   operation:
     required: true
     aliases: [ command ]
-    choices: [ create, comment, edit, fetch, transition ]
+    choices: [ create, comment, edit, fetch, transition, block_link ]
```

Comment:
make a note in the description that block_link is only available since 2.1 as users of older versions looking at the docs won't think they can use it.